gravitee-am-gateway/gravitee-am-gateway-handler/gravitee-am-gateway-handler-scim/src/main/java/io/gravitee/am/gateway/handler/scim/service/impl/UserServiceImpl.java

```diff
@@ -348,9 +359,12 @@ public Single<User> update(String userId, User user, String idp, String baseUrl,
                                         .doOnError(error -> auditService.report(AuditBuilder.builder(UserAuditBuilder.class).principal(principal).user(existingUser).type(EventType.USER_UPDATED).throwable(error))));
                             }));
                 })
-                .map(user1 -> UserMapper.convert(user1, baseUrl, false))
+                .map(user1 -> {
+                    createPasswordHistory(domain, user1, rawPassword, principal);
```

Comment:
As it is an update, in order to reject the update if the password already exist we should do the same kind of call as the one made during resetPassword https://github.com/gravitee-io/gravitee-access-management/pull/2188/files#diff-c09299b3f56fdb15db97bbff8867279a03f6f636d5dcfcaabca730aade600366R300